### ISSUE TITLE
SelectOneRadio: improve ARIA accessibility of custom layout

### ISSUE THREAD

**Issue by strefilov** [primefaces/primefaces#8180]
Problem with screen reader JAWS or NVDA
jaws(screen reader ) can't read element radioButton and read as 1 from 1 without label 

**Reproducer**
https://www.primefaces.org/showcase/ui/input/oneRadio.xhtml?jfwid=574c3

**Environment:**
 - PF Version: 11
 - JSF + version: Mojarra 2.2.20 or MyFaces 2.3.9_
 - Affected browsers:  ALL

**To Reproduce**
1. run screen reader 
2. Click on Red radioButton 
3. listen reader 
4. select Green by arrow
5. listen reader 

**Expected behavior**
screen reader can read information from aria-label but input doesn't have it

**Example XHTML**
```html
[<h:form id="frmTest">
<p:outputPanel id="customPanel" style="margin-bottom:10px">
            <p:selectOneRadio id="customRadio" value="#{radioView.color}" layout="custom">
                <f:selectItem itemLabel="Red" itemValue="Red"/>
                <f:selectItem itemLabel="Green" itemValue="Green"/>
                <f:selectItem itemLabel="Blue" itemValue="Blue"/>
            </p:selectOneRadio>

            <h:panelGrid columns="3" cellpadding="5">
                <p:radioButton id="opt1" for="customRadio" itemIndex="0"/>
                <h:outputLabel for="opt1" value="Red"/>
                <p:spinner/>

                <p:radioButton id="opt2" for="customRadio" itemIndex="1"/>
                <h:outputLabel for="opt2" value="Green"/>
                <p:inputText/>

                <p:radioButton id="opt3" for="customRadio" itemIndex="2"/>
                <h:outputLabel for="opt3" value="Blue"/>
                <p:calendar/>
            </h:panelGrid>
        </p:outputPanel>
```

```

**Comment by jepsar** [comment]
Can be assigned to me.

**Comment by melloware** [comment]
Done!

**Comment by jepsar** [comment]
@strefilov PR is ready for review.

**Comment by strefilov** [comment]
it works for NVDA but jaws can't read this solution. jaws(reader) can work with "aria-lable" . 
I recommended add property aria-lable for p:radioButton. Developer will be able to select solution .

**Comment by jepsar** [comment]
@strefilov can you please reference documentation.

**Comment by strefilov** [comment]
![image](https://user-images.githubusercontent.com/7150149/147451391-0e099939-110f-48c5-988a-ff16d57acb16.png)

**Comment by strefilov** [comment]
jaws not read it

**Comment by jepsar** [comment]
@strefilov can you please reference documentation on this? I based my changes on https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html

Would adding `aria-labelledby` to the radio button help? That's what we use in general for components.

**Comment by strefilov** [comment]
https://www.w3.org/TR/wai-aria-1.1/#radio 
https://www.w3.org/TR/wai-aria-1.2/#radio
to see Inherited States and Properties:

**Comment by strefilov** [comment]
please add support : 
aria-label

**Comment by jepsar** [comment]
@strefilov I've updated the PR to make the labels point to the original inputs. That is the same as the other, non-custom, SelectOneRadio variants (assuming that those are working for you). See https://github.com/primefaces/primefaces/pull/8194/commits/0da7ea14914a2e534cc191a04376f7f89fd17400

Can you please try the the updated PR.

**Comment by strefilov** [comment]
it works ok!!

**Comment by jepsar** [comment]
@strefilov Great! Thank you for reporting and testing.

**Comment by jepsar** [comment]
@strefilov Please don't close issues with open PRs.

**Comment by strefilov** [comment]
sorry

**Comment by jepsar** [comment]
@strefilov Can you please reopen the issue?
